package.json

```diff
@@ -47,6 +47,7 @@
     "style-loader": "^0.13.1",
     "stylelint": "^7.13.0",
     "stylelint-config-standard": "^16.0.0",
+    "uglifyjs-webpack-plugin": "^1.2.2",
```

Comment:
AFAICT this should be <1.2.0. At least 1.2.1(?) has weback 4.x as peer dependency.

Reply:
Webpack pulls it in by default so if it's < 1.2.0 then it's redundant.

Reply:
The peer dependency of `uglifyjs-webpack-plugin@1.2.2` are `"webpack": "^2.0.0 || ^3.0.0 || ^4.0.0"`
So i think it should be fine.

`webpack@3` pulling `"uglifyjs-webpack-plugin": "^0.4.6"` wich is the version with the problem.

Do you think it is somehow related?
https://twitter.com/wSokra/status/967852475918274561

Reply:
Without adding this line it also works for me so I'd prefer to not include this. For me it pulls in 1.1.6 without it and that works.

Reply:
@ekohl i'm not sure I get it, can you explain please?

Reply:
@sharvit I tend to prefer as little specific pins as possible. If this isn't needed, I'd prefer not to add it.

Reply:
Looking back I was wrong. We explicitly require it our config so we should depend on it. It also looks like it needs at least a certain version (0.4.6 is being pulled in on slaves and doesn't work).

Reply:
GH-5289